pkg/bearertoken/context.go

```diff
@@ -12,30 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package spanstore
+package bearertoken
 
 import "context"
 
-type contextKey string
+type contextKeyType string
 
-// BearerTokenKey is the string literal used internally in the implementation of this context.
-const BearerTokenKey = "bearer.token"
-const bearerToken = contextKey(BearerTokenKey)
+const (
+	// Key is the key name for the bearer token context value.
+	Key = "bearer.token"
```

Comment:
`Key` is used outside of this package [here](https://github.com/jaegertracing/jaeger/blob/6685913c38a7168bc32c6a8cfb5a37546029f390/plugin/storage/grpc/shared/grpc_client.go#L76).

Reply:
Addressed in 696cb5f.